code/modules/mob/living/carbon/human/species/golem.dm

```diff
@@ -9,12 +9,7 @@
 	inherent_biotypes = MOB_HUMANOID | MOB_MINERAL
 	dies_at_threshold = TRUE
 	speed_mod = 2
-	brute_mod = 0.45 //55% damage reduction
-	burn_mod = 0.45
-	tox_mod = 0.45
-	clone_mod = 0.45
-	brain_mod = 0.45
-	stamina_mod = 0.45
+	armor = 55
```

Comment:
All of these values are the same as before, despite appearing different--we handle these with a single `armor` value now.